cdap-docs/_common/_themes/cdap/static/cdap.css_t

```diff
@@ -301,13 +301,13 @@ p.admonition-title:after {
 }
 
 p.rubric {
-    background-color: #E0E0E0;
-    font-size: 120%;
-    font-weight: normal;
+    background-color: #EDEDED; 
     color: #111;
+    font-size: 110%; 
+    font-weight:normal;
     margin: 30px 0px 10px 0px;
-    padding: 5px 0 5px 15px;
-    text-shadow: 0px 1px 0 white
+    padding: 5px 0px  5px  15px;
```

Comment:
`padding: 5px 0 5px 15px` is considered better than `0px`. If you pass the style through a minifier it will probably minify it to `0` instead of `0px`.